src/BasicRum/Release.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\BasicRum;
+
+use DateTime;
+
+class Release
+{
+    private $releasses;
+
+    public function __construct(\App\Repository\ReleasesRepository $releases)
+    {
+        $this->releases = $releases;
+    }
+
+    /**
+     * @param string $date1 Start date
+     * @param string $date2 End date
+     *
+     * @return array [description]
+     */
+    public function getAllReleasesBetweenDates(string $date1, string $date2): array
```

Comment:
Please give more descriptive variables names $date1 and $date2 is not bad but what stops us to have $startData and $endDate ?